tests/TypeTests.hs

```diff
@@ -46,7 +46,7 @@ import GraphQL.Value (Value(..))
 data DogCommandEnum = Sit | Down | Heel
 
 instance GraphQLEnum DogCommandEnum where
-  enumValues = map unsafeMakeName ["SIT", "DOWN", "HEEL"]
+  enumValues = map makeName ["SIT", "DOWN", "HEEL"]
```

Comment:
Done, but had to re-introduce a Proxy hack because GHC isn't clever enough to resolve the type variable `a` in the type class otherwise.